src/citrine/resources/predictor_evaluation_execution.py

```diff
@@ -0,0 +1,118 @@
+"""Resources that represent both individual and collections of workflow executions."""
+from functools import lru_cache
+from typing import Optional, Set
+from uuid import UUID
+
+from citrine._rest.collection import Collection
+from citrine._rest.resource import Resource
+from citrine._serialization import properties
+from citrine._session import Session
+from citrine.informatics.predictor_evaluation_result import PredictorEvaluationResult
+
+
+class PredictorEvaluationExecution(Resource['PredictorEvaluationExecution']):
+    """[ALPHA] The execution of a PredictorEvaluationWorkflow.
+
+    Parameters
+    ----------
+    uid: str
+        Unique identifier of the workflow execution
+    project_id: str
+        Unique identifier of the project that contains the workflow execution
+    workflow_id: str
+        Unique identifier of the workflow that was executed
+    version_number: int
+        Integer identifier that increases each time the workflow is executed.
+        The first execution has version_number = 1.
+
+    """
+
+    _response_key = 'WorkflowExecutions'
+
+    uid = properties.UUID('id')
+    project_id = properties.UUID('project_id', deserializable=False)
+    workflow_id = properties.UUID('workflow_id', deserializable=False)
+    version_number = properties.Integer("version_number")
```

Comment:
Does `version_number` make sense here in the way that it is used for design workflows?  I think we might want to ax it.

Reply:
I'll remove it.